### PR TITLE
[BXMSPROD-1354] Jenkinsfile.nightly updated

### PR DESCRIPTION
**Pull request by Ginxo**
**Thank you for submitting this pull request**

**JIRA**: https://issues.redhat.com/browse/BXMSPROD-1354

[link](https://examle.com)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* https://github.com/kiegroup/kogito-tooling/pull/557
* https://github.com/kiegroup/kie-jenkins-scripts/pull/1035

### CODE REVIEW

**Review by domhanak — APPROVED** [review]

File: Jenkinsfile.nightly [hunk 1]
```diff
@@ -72,7 +88,8 @@ pipeline {
 {
     \"npmRegistry\": \"${NPM_REGISTRY_PUBLISH_URL}",
     \"versions\": {
-        \"kogito-tooling\": \"${env.CURRENT_KOGITO_TOOLING_VERSION}\"
+        \"kogito-tooling\": \"${env.CURRENT_KOGITO_TOOLING_VERSION}\",
+        \"kogito-editors-java\": \"${env.kogito_editors_java_VERSION}\"
```
Comment (jstastny-cz) [inline Jenkinsfile.nightly]: wouldn't it be better to keep upper case as rest of the variables?

File: Jenkinsfile.nightly [hunk 2]
```diff
@@ -128,18 +143,19 @@ def getBuildConfigMap(def buildConfigPathFolder, def buildConfigAdditionalVariab
 
 def buildProject(String project, Map<String, Object> buildConfig, String defaultGroup = "kiegroup") {
     println "[INFO] Building project ${project}"
+    if("kiegroup/kogito-tooling" == project) {
```
Comment (jstastny-cz) [inline Jenkinsfile.nightly]: whitespace missing after `if`

File: Jenkinsfile.nightly [hunk 3]
```diff
@@ -128,18 +143,19 @@ def getBuildConfigMap(def buildConfigPathFolder, def buildConfigAdditionalVariab
...
         if (fileExists("./pom.xml")) {
+            pmebuild.executePME(project, pmebuild.getProjectConfiguration(project, buildConfig), "${env.PME_CLI_PATH}", settingsXMLId, [:])
             def pom = readMavenPom file: 'pom.xml'
             env["${name}-VERSION".replace("-", "_")] = pom.version ?: pom.parent.version
         }
-        pmebuild.executeBuildScript("${group}/${name}", buildConfig, "kogito-tooling-prod")
+        pmebuild.executeBuildScript(project, buildConfig, settingsXMLId)
     }
-}
+}
```
Comment (jstastny-cz) [inline Jenkinsfile.nightly]: This whitespace change might not be expected.

**Review by jstastny-cz — APPROVED** [review]
Looks good, just minor comments.

File: Jenkinsfile.nightly (hunk 2 above)
Comment (Ginxo) [reply Jenkinsfile.nightly]: done

File: Jenkinsfile.nightly (hunk 1 above)
Comment (Ginxo) [reply Jenkinsfile.nightly]: done

**Pull request merged**